streaming/http-to-mysql/README.adoc

```diff
@@ -63,12 +63,48 @@ Welcome to the Spring Cloud Data Flow shell. For assistance hit TAB or type "hel
 dataflow:>version
 1.0.0.BUILD-SNAPSHOT
 ```
+
 +
-. Create the stream
+. Patch the `jdbc` sink with `mysql` driver
+
+- build the https://github.com/spring-cloud/spring-cloud-stream-app-starters#spring-cloud-stream-app-starters[spring-cloud-stream-app-starters] project
+- cd `jdbc-sink-kafka`
+- modify `pom.xml` to include `mysql` driver (_remove `hsqldb`_)
++
+[source,xml]
+----
+      <dependency>
+        <groupId>org.hsqldb</groupId>
+        <artifactId>hsqldb</artifactId>
+        <version>2.3.3</version>
+        <scope>exclude</scope>
```

Comment:
There is no such a `scope` in [Maven|https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html].
How about *remove* this `hsqldb` dependency or comment out in the XML style?

(I can do that on merge if only a concern)